lambda/dynamo-api/src/client/DynamoClient.ts

```diff
@@ -0,0 +1,31 @@
+import {DynamoDBClient, PutItemCommand, PutItemCommandOutput} from "@aws-sdk/client-dynamodb";
+import {marshall, unmarshall} from "@aws-sdk/util-dynamodb";
+import {OnboardingTableItem} from "../@Types/OnboardingTableItem";
+
+class DynamoClient {
+    private dynamodb: DynamoDBClient;
+    private readonly tableName: string;
+
+    constructor(tableName: string) {
+        this.dynamodb = new DynamoDBClient({ region: "eu-west-2" });
```

Comment:
Can we move the region to env file? It is less likely to change but just in case.

Reply:
It would be an env var from the SAM template for a lambda but that can definitely be a configurable value.

Reply:
In fact, AWS_REGION is available by default so we don't even have to configure it ...